dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -45,11 +45,17 @@ object TankGenerator {
   private def distributeAssets(accounts: List[PrivateKeyAccount], assets: List[String]): Unit = {
     println(s"Distributing assets to accounts... ")
     val amountPerUser = settings.assets.quantity / 4 / accounts.length
+
+    def assetBalanceIsNotEnough(ac: PrivateKeyAccount, as: String): Boolean = {
+      services.node.getBalance(ac.getAddress, as) < (settings.defaults.maxOrdersPerAccount * settings.defaults.minimalOrderAmount * 2) &&
```

Comment:
`(settings.defaults.maxOrdersPerAccount * settings.defaults.minimalOrderAmount * 2)` is a constant that will be recalculated anytime we use `assetBalanceIsNotEnough` method

Reply:
ok, moved outside